content/zh/docs/reference/command-line-tools-reference/kubelet.md

```diff
@@ -63,50 +53,2109 @@ kubelet [flags]
+      -->
+      <警告：beta 功能> 设置容器可以存在的最大容器日志文件数，必须不小于 2。此参数只能与 --container-runtime=remote 参数一起使用。（默认值为 5）。（已弃用：在 --config 指定的配置文件中进行设置。有关更多信息，请参阅 https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/。）
+      </td>
+    </tr>    
+  
+    <tr>
+       <td colspan="2">--container-log-max-size string</td>
+    </tr>
+    <tr>
+      <td></td><td style="line-height: 130%; word-wrap: break-word;">
+      <!--
+      <Warning: Beta feature> Set the maximum size (e.g. 10Mi) of container log file before it is rotated. This flag can only be used with --container-runtime=remote. (default "10Mi") (DEPRECATED: This parameter should be set via the config file specified by the Kubelet's --config flag. See https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information.)
+      -->
+      <警告：beta 功能> 设置容器日志文件在滚动生成新文件时之前的最大值。此参数只能与 --container-runtime=remote 参数一起使用。（默认值为 “10Mi”）（已弃用：在 --config 指定的配置文件中进行设置。有关更多信息，请参阅 https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/。）
+      </td>
+    </tr>
+
+    <tr>
+       <td colspan="2">--container-runtime string</td>
+    </tr>
+    <tr>
+      <td></td><td style="line-height: 130%; word-wrap: break-word;">
+      <!--
+      The container runtime to use. Possible values: 'docker', 'remote', 'rkt(deprecated)'. (default "docker")
+      -->
+      容器类型。目前支持 ‘docker’、‘remote’、‘rkt(已弃用)’。（默认值为 ‘docker’）。
+      </td>
+    </tr>
+
+     <tr>
+       <td colspan="2">--container-runtime-endpoint string</td>
+    </tr>
+    <tr>
+      <td></td><td style="line-height: 130%; word-wrap: break-word;">
+      <!--
+      [Experimental] The endpoint of remote runtime service. Currently unix socket endpoint is supported on Linux, while npipe and tcp endpoints are supported on windows.  Examples:'unix:///var/run/dockershim.sock', 'npipe:////./pipe/dockershim' (default "unix:///var/run/dockershim.sock")
+      -->
+      [实验性特性] 容器运行时的远程服务 endpoint。目前，在 Linux 系统上支持的类型包括 unix socket，在 Windows 系统上支持的类型包括 npipe 和 tcp endpoint。例如：‘unix:///var/run/dockershim.sock’、‘npipe:////./pipe/dockershim’。（默认值为 “unix:///var/run/dockershim.sock”）
+      </td>
+    </tr>
+
+     <tr>
+       <td colspan="2">--containerd string</td>
+    </tr>
+    <tr>
+      <td></td><td style="line-height: 130%; word-wrap: break-word;">
+      <!--
+      containerd endpoint (default "/run/containerd/containerd.sock") (DEPRECATED: This is a cadvisor flag that was mistakenly registered with the Kubelet. Due to legacy concerns, it will follow the standard CLI deprecation timeline before being removed.)
```

Comment:
```suggestion
      containerd endpoint (default "/run/containerd/containerd.sock") (DEPRECATED: This is a cadvisor flag that was mistakenly registered with the Kubelet. Due to legacy concerns, it will follow the standard CLI deprecation timeline before being removed.)
      -->
```